src-built-in/components/appCatalog2/src/stores/storeActions.js

```diff
@@ -31,8 +31,8 @@ let FDC3Client;
 let appd;
 
 function initialize(done = Function.prototype) {
-	FSBL.Clients.ConfigClient.getValue({ field: "finsemble.FDC3Server" }, function (err, FDC3Server) {
-		FDC3Client = new FDC3({ url: FDC3Server });
+	FSBL.Clients.ConfigClient.getValue({ field: "finsemble.appDirectoryEndpoint" }, function (err, appDirectoryEndpoint) {
```

Comment:
My thoughts, why not use async/await here to avoid having a whole logic in a callback function? There is going to be no way to write unit test in the future, if we wanted to.

Unless the ConfigClient's getValue does not return a promise, correct me if I'm wrong.